ethpm/_utils/backend.py

```diff
@@ -13,51 +13,25 @@
     get_ipfs_backend_class,
 )
 from ethpm.backends.registry import RegistryURIBackend
-from ethpm.exceptions import CannotHandleURI
 from ethpm.typing import URI
 
-URI_BACKENDS = [
+logger = logging.getLogger("ethpm.utils.backend")
+
+ALL_URI_BACKENDS = [
```

Comment:
orthogonal but probably worth adopting a registry pattern similar to what was done here: https://github.com/ethereum/trinity/pull/586/files#diff-bd2ae038cb9e0fb653d90082dcf5ddfcR42